alot/helper.py

```diff
@@ -625,3 +625,9 @@ def email_as_string(mail):
                            as_string, flags=re.MULTILINE)
 
     return as_string
+
+
+def get_env(env_name, fallback):
+    """ Gets environment variable and returns fallback if unset or empty """
+    env = os.environ.get(env_name)
+    return env if env else fallback
```

Comment:
How is this function any better than a direct call like `os.environ.get(env_name, fallback)`?

Reply:
This could also be replaced with a simple `get_env = os.environ.get` and the tests should still pass. But then we could just replace it altogether and ditch the tests for get_env.

Reply:
I should name the function better and add a comment - maybe `get_xdg_env`. The point is that `os.environ.get('env, fallback)` returns `''`, if the variable is set, but empty. But in the case of an empty `XDG_*` variable, it requires the fallback value, otherwise in all the cases where I use `get_env` the paths are wrong.

Reply:
Ok I see. I will now approve, you can decide on your own if you want to take the comments in the tests into account or not.

Reply:
Cool - I'll try to simplify the tests with the help of your comments.